cwltool/utils.py

```diff
@@ -19,9 +24,93 @@ def get_feature(self, feature):  # type: (Any, Any) -> Tuple[Any, bool]
             return (t, True)
     for t in reversed(self.hints):
         if t["class"] == feature:
+            if "requirement" in t and t["requirement"] is True:
+                return(t, True)
```

Comment:
Yes, Originally requirements were compulsory checked while hints were not. But as we have added `default-container` on windows, We are adding it as lowest priority Hints. So by default, it isn't checked. So to enforce checking even if a container is in hints(i.e default container) I added this flag. would `get_feature_and_requirement` work as new name.